### PR TITLE
SHM6: Class Island - implementation

### LINKED ISSUE #6: SHM6: Class Island - implementation
Task 3 - Part 2

Write the Island class that will have the Coordinates position_ variable and the appropriate getter.

### PR DESCRIPTION
**Pull request by NG90**
resolves #6

### CODE REVIEW

File: shm/island.cpp [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#include "island.hpp"
+
+Island::Island(size_t position_X, size_t position_Y)
```
Comment (DarkAriesX) [inline shm/island.cpp:3]: I think better is to use `int` to coordinates, because they can be negative

Comment (NG90) [reply shm/island.cpp:3]: ok

Comment (NG90) [reply shm/island.cpp]: done

File: shm/island.hpp [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#pragma once
+#include "coordinates.hpp"
+
+class Island {
+public:
+    Island(int, int);
```
Comment (NG90) [inline shm/island.hpp:6]: add argument names

**Pull request merged**